### PR TITLE
RDoc-2631 [Python] Client API > Session > Storing entities [Replace C# samples]

### PR DESCRIPTION
**Pull request by reebhub**
Related issue:
[RDoc-2631](https://issues.hibernatingrhinos.com/issue/RDoc-2631/Python-Client-API-Session-Storing-entities-Replace-C-samples)

### CODE REVIEW

File: Documentation/5.4/Raven.Documentation.Pages/client-api/session/storing-entities.python.markdown [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+
+Stores the entity in a session, then extracts the ID from the entity or if not available generates a new one.
+
+{CODE:python store_entities_1@ClientApi\Session\StoringEntities.py /}
+
+| Parameters | | |
+| ------------- | ------------- | ----- |
+| **entity** | object | Entity that will be stored |
+| **change_vector** | string | Entity change vector, used for concurrency checks (`null` to skip check) |
+| **id** | string | Entity will be stored under this ID, (`null` to generate automatically) |
```
Comment (poissoncorp) [inline Documentation/5.4/Raven.Documentation.Pages/client-api/session/storing-entities.python.markdown:15]:
```suggestion
| **key** | str | Entity will be stored under this ID, (`None` to generate automatically) |
```

**Review by poissoncorp — CHANGES_REQUESTED** [review]
similar to https://github.com/ravendb/docs/pull/1771

File: Documentation/5.4/Raven.Documentation.Pages/client-api/session/storing-entities.python.markdown (hunk 1 above)
Comment (reebhub) [reply Documentation/5.4/Raven.Documentation.Pages/client-api/session/storing-entities.python.markdown:15]: done

File: Documentation/5.4/Raven.Documentation.Pages/client-api/session/storing-entities.python.markdown [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+## Syntax
+
+Stores the entity in a session, then extracts the ID from the entity or if not available generates a new one.
+
+{CODE:python store_entities_1@ClientApi\Session\StoringEntities.py /}
+
+| Parameters | | |
+| ------------- | ------------- | ----- |
+| **entity** | object | Entity that will be stored |
+| **change_vector** | string | Entity change vector, used for concurrency checks (`null` to skip check) |
```
[the remark this answers is not in the document]
Comment (reebhub) [reply Documentation/5.4/Raven.Documentation.Pages/client-api/session/storing-entities.python.markdown:14]: done

**Review by poissoncorp — APPROVED** [review]
ready to go

**Pull request merged**